### DISCUSSION TITLE
Custom Page -  Table Action  - Dispatch Modal

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#11032] [Help]
Hi, i  launch custom modal from action in table row, it s ok but how can pass params and retrieve it in modal (example : employee record) like below :
` ->action(function (Employee $record): void {
       $this->dispatch('open-modal', id: 'employee-view-modal', employee : $record->getKey());
}),`

Thanks

**Comment by leandrocfe** [comment]
Why not a custom action+infolist?

```php
$table
...
->actions([
    Tables\Actions\EditAction::make(),
    Tables\Actions\Action::make('customView')
    ->infolist([
        TextEntry::make('employee_name'),
        ...
    ])
])
```

**Reply by warsemann** [reply]
I want a custom view, it s not possible with infolist ?
